spring-restdocs-asciidoctor/src/main/java/org/springframework/restdocs/asciidoctor/RestDocsExtensionRegistry.java

```diff
@@ -28,8 +30,15 @@
 
 	@Override
 	public void register(Asciidoctor asciidoctor) {
-		asciidoctor.javaExtensionRegistry()
-				.preprocessor(new DefaultAttributesPreprocessor());
+		final JavaExtensionRegistry registry = asciidoctor.javaExtensionRegistry();
+		registry.preprocessor(new DefaultAttributesPreprocessor());
+
+		RubyExtensionRegistry rubyExtensionRegistry = asciidoctor.rubyExtensionRegistry();
+		rubyExtensionRegistry
+				.loadClass(RestDocsExtensionRegistry.class
+						.getResourceAsStream("/extensions/srd-include-block-macro.rb"))
+				.blockMacro("srd", "RestDocsBlockMacro");
```

Comment:
I don't think the macro should be called `srd` as that would seem to imply the REST Docs will only ever have a single block macro. There doesn't seem to be a tradition of trying to namespace macro names in Asciidoctor so I'm tempted to go with `operation` instead.